pkg/lib/fieldgroups/database/database_validator.go

```diff
@@ -28,8 +31,38 @@ func (fg *DatabaseFieldGroup) Validate(opts shared.Options) []shared.ValidationE
 		ca = fg.DbConnectionArgs.Ssl.Ca
 	}
 
+	sslrootcertTmpPath := fg.DbConnectionArgs.SslRootCert
+	if fg.DbConnectionArgs.SslMode == "verify-full" || fg.DbConnectionArgs.SslMode == "verify-ca" {
```

Comment:
Can't `DbConnectionArgs` be `nil`? If it can then this might `panic`

Reply:
We reference `DbConnectionArgs` with doing a check whether it's `nil` or not, so I just assumed that it's been correctly initialized..